src/pages/billing/Project.js

```diff
@@ -53,20 +54,29 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
   const [selectedBilling, setSelectedBilling] = useState()
   const [tab, setTab] = useState(query.tab || 'workspaces')
 
+  const [sort, setSort] = useState({ field: 'email', direction: 'asc' })
+
+
   const signal = Utils.useCancellation()
 
+  const adminCanEdit = _.filter(({ roles }) => _.includes(billingRoles.owner, roles), projectUsers).length > 1
+
   const tabToTable = {
     workspaces: div({ style: { flexGrow: 1, width: '100%' } }, [
       _.flow(
-        _.filter(response => response.workspace.namespace === projectName),
+        _.filter({ workspace: { namespace: projectName } }),
         _.map(({ workspace }) => h(WorkspaceCard, { workspace, key: workspace.workspaceId }))
       )(workspaces)
     ]),
     users: h(Fragment, [
       h(NewUserCard, {
         onClick: () => setAddingUser(true)
-      }),
-      div({ style: { flexGrow: 1 } },
+      }, [
+        icon('plus-circle', { size: 14 }),
+        div({ style: { marginLeft: '0.5rem' } }, ['Add User'])
+      ]),
+      h(MemberCardHeaders, { sort, onSort: setSort }),
+      div(
```

Comment:
When reasonable, I don't like to have elements with things in them declared on their own line; can you bring the `_.map` up?

Reply:
I personally find the indentation of this makes it more readable, is this a codebase pattern? If so, I can adapt it to that, but want to make sure its a codebase pattern first.

Reply:
If we don't have a codebase pattern there, maybe it makes sense to just catch up real quick about that first before I make the change?

Reply:
It is a pattern in the codebase, yes. For example, this `div` previously had its props object on the same line as the function call; now, with now props, I'd want the children on the same line.

Reply:
Done